modules/serverless-enabling-tls-internal-traffic.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * /serverless/security/serverless-config-tls.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-enabling-tls-internal-traffic_{context}"]
+= Enabling TLS authentication for internal traffic
+
+{ServerlessProductName} supports TLS edge termination by default, so HTTPS traffic from end users is encrypted. However, internal traffic behind the OpenShift route is forwarded to applications by using plain data. By enabling TLS for internal traffic, the traffic sent between components is encrypted, which makes this traffic more secure.
+
+:FeatureName: Internal TLS encryption support
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have installed the {ServerlessOperatorName} and Knative Serving.
+* You have installed the OpenShift (`oc`) CLI.
+* If you want to enable internal TLS with a Service Mesh integration, you must enable Service Mesh with mTLS.
+
+.Procedure
+
+. Create a Knative service that includes the `internal-encryption: "true"` field in the spec:
++
+[source,yaml]
+----
+...
+spec:
+  config:
+    network:
+      internal-encryption: "true"
+...
+----
+
+. Restart the activator pods in the `knative-serving` namespace to mount the certificates:
```

Comment:
```suggestion
. Restart the activator pods in the `knative-serving` namespace to load the certificates:
```

To be exact, activator loads the certificates instead of `mount` so it is better to use `load` (or `read`).
(Sorry I wrote mount in the draft doc.)

Reply:
great, thank you!